CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ CHANGELOG
 - SlurmQueues.Networking.PlacementGroup.Enabled alone will now create a unique managed placement for each compute resource instead of a single managed placement group for all compute resources
 
 **BUG FIXES**
+- Fix validation of `filters` parameter in `ListClusterLogStreams` command: now validation fails with a proper
```

Comment:
What about: Fix validation of `filters` parameter in `ListClusterLogStreams` command to fail when incorrect filters are passed.

Reply:
make sense, will fix